retroshare-gui/src/gui/FileTransfer/SharedFilesDialog.cpp

```diff
@@ -74,7 +74,7 @@
 #define IMAGE_COLLOPEN       ":/icons/collections.png"
 #define IMAGE_EDITSHARE      ":/images/edit_16.png"
 #define IMAGE_MYFILES        ":/icons/svg/folders1.svg"
-#define IMAGE_REMOVE         ":/images/deletemail24.png"
+#define IMAGE_REMOVE         ""
```

Comment:
if the deletemail24.png image is removed, then you should probably also remove IMAGE_REMOVE, so that you'll see if it is still used or not.